source/agora/consensus/Quorum.d

```diff
@@ -0,0 +1,1118 @@
+/*******************************************************************************
+
+    Contains the quorum generator algorithm.
+
+    Note: If you use AscendingQuadratic for too many nodes, it will
```

Comment:
> Also this could use more actual documentation.

Well it's a draft. If half the code gets thrown away I'd hate to throw away the documentation too. I'll add it as soon as it's finalized.